pkg/pillar/dpcreconciler/linux.go

```diff
@@ -1505,9 +1578,22 @@ func (r *LinuxDpcReconciler) getIntendedACLs(
 		TargetOpts:  []string{"--set-mark", iptables.ControlProtocolMarkingIDMap["in_dhcp"]},
 		Description: "Mark ingress DHCP traffic",
 	}
+	// XXX allow kubernetes DNS replies from external server. Maybe there is
+	// better way to setup this, like using set-mark for outbound kubernetes DNS queires.
+	markDNS := iptables.Rule{
+		RuleLabel:   "Incoming DNS traffic",
```

Comment:
Are this DNS replies? label doesn't read like replies.

Reply:
Yes, this is for DNS replies to the kube dns service query.

Reply:
Can't we use conntrac state for UDP here?

Reply:
For the current single node, yes the conntrack should work, but for later multi-node cluster, one node has the coreDNS service, will need to allow others to access.

Reply:
But that is a completely different thing; enabling access to a DNS server, which would need a separate ACL.

Here it seems to be a DNS client needing access to an external server.

Reply:
Updated label to refer to this traffic as "DNS replies".
I belive we can improve this part but we need to think it through. For this initial PoC K3s-EVE integration I think it is good enough.